api/src/pcapi/notifications/push/backends/batch.py

```diff
@@ -87,11 +89,14 @@ def make_post_request(api: BatchAPI) -> None:
         make_post_request(BatchAPI.ANDROID)
         make_post_request(BatchAPI.IOS)
 
-    def send_transactional_notification(self, notification_data: TransactionalNotificationData) -> None:
+    def send_transactional_notification(
+        self, notification_data: TransactionalNotificationData, has_asynchronous_retry: bool = False
+    ) -> None:
         def make_post_request(api: BatchAPI) -> None:
             try:
                 response = requests.post(
                     f"{settings.BATCH_API_URL}/1.1/{api.value}/transactional/send",
+                    disable_synchronous_retry=has_asynchronous_retry,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Du coup j'aurais suggéré `can_retry_asynchronously` 
`has_stuff` je sais jamais trop si c'est `contient_stuff`, `a_déjà_stuff_précédement`, `possède_stuff`

`can_do_stuff` est plus direct

---

Eventuellement `has_already_retried_asynchronously`

Reply:
has_asynchronous_retry est peut-être mal nommé alors, ce serait : `is_in_automatic_asynchronous_retry_workflow`

Reply:
J'ai mis `can_be_asynchronously_retried`